ui/src/parts/lightbox.tsx

```diff
@@ -177,22 +178,28 @@ export const Lightbox = ({ images, defaultImageIdx }: LightboxProps) => {
           iconProps={{ iconName: 'Cancel', style: styles.icon }}
         />
       </div>
-      <div className={css.content} style={{ height: `calc(100% - ${HEADER_HEIGHT + FOOTER_HEIGHT}px)` }}>
-        <img className={css.img} alt={title} src={getImageSrc(images[activeImageIdx])} />
+      <div className={css.content} style={{ height: `calc(100% - ${HEADER_HEIGHT + FOOTER_HEIGHT}px)` }} onClick={() => lightboxB(null)}>
+        <img className={css.img} alt={title} src={images[activeImageIdx].path} onClick={ev => ev.stopPropagation()} />
         {isGallery &&
           <>
             <Fluent.ActionButton
               styles={iconStyles}
               className={css.iconStylesRootArrow}
               style={{ left: 0 }}
-              onClick={() => setActiveImageIdx(prevIdx => prevIdx === 0 ? images.length - 1 : prevIdx - 1)}
+              onClick={ev => {
+                ev.stopPropagation()
+                setActiveImageIdx(prevIdx => prevIdx === 0 ? images.length - 1 : prevIdx - 1)
+              }}
```

Comment:
I've got rid of `ev.stopPropagation()` line. Is the separate function still necessary?

Reply:
Leaving up to you. The general rule of thumb is: simple one-liner is fine to be inlined, otherwise use a separate function so that we do not mess JS and HTML together more than necessary.